### ISSUE TITLE
Create a Blog page

### ISSUE THREAD

**Issue by hotRedPanda1** [inland-empire-software-development/spark#48]
**Is your feature request related to a problem? Please describe.**
I like reading through blogs related to work-life in the tech industry. 
 
**Describe the solution you'd like**
(1) There should be various articles posted after a severe review and check to see that each article has met the specific guidelines of submission. 
(2) We need an article submission page that will allow writers to submit their article for review before it is posted. 

**Describe alternatives you've considered**
There must be a page of clear, concise article submission guidelines for writers to follow.

**Additional context**
(1) There must be a limit to how many articles are accepted and posted every month. 
(2) An article that meets the requirements yet cannot be posted for a specific month may or may not be recycled or saved and utilized in a later edition. 
(3) All articles that are considered for publication become the property of IESD or SPARK.

**Comment by lloan** [comment]
@hotRedPanda1 This is great, but we actually could use this on www.iesd.com - if you visit the site, one of the first components you see is the `newsroom`, but we don't have an actual page dedicated to the newsroom, which I think would be very helpful, specially as we have people that actually want to start writing. Maybe this issue might be better fitted for this repo: https://github.com/inland-empire-software-development/main - You could lead the overall functional design of it and I could get you a developer or two to work with to create that page. It's a high priority item that we really need.

**Comment by hotRedPanda1** [comment]
I would love to head the newsroom project with your guidance and patience.
However, since that is a high priority, do you want me to work on that
instead of the Spark project? Or do you want me to focus on both projects?

**Comment by lloan** [comment]
@hotRedPanda1 Definitely. As we talked about earlier today at the meeting, we would like to get this project out the door too as it would be very useful for the community to have. We have a lot of different kinds of topics to write about: updates, student journeys about their learning, tech articles, etc. 

The way that you play in to this is that you bring your user experience and knowledge about learning. 

Think about the following questions:
- How do you think the newsroom should flow.
- Who is our target user
- Is the user able to get to where they need to get.
- What do we want the user to get out of using the newsroom?
- How do we get users to come back>
- How can users interact with our articles and/or should they?
- What is the ideal length?
- Tools for article writers? 
- User experience for the user.
- User experience for the writer. 
- What kind of pages do we foresee for this system? author pages, category pages, filter page (search), etc.
